client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -55,206 +59,184 @@ describe( 'BackupDatePicker', () => {
 	// --- DATE DISPLAY ---
 
 	test( "Shows 'Yesterday' as the previous date if the current date is today", () => {
-		const today = moment();
-
-		const picker = shallow( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
+		Date.now = jest.fn().mockReturnValue( new Date( '2020-12-04T12:00:00.000Z' ) );
 
-		const previousDate = picker.find(
-			'.backup-date-picker__select-date--previous .backup-date-picker__display-date'
+		const today = moment();
```

Comment:
Perhaps this should do it instead of needing to mock `Date.now` or `moment`?

```suggestion
		const today = moment( '2020-12-04 12:00 +0000' );
```